src/MainWindow.cpp

```diff
@@ -425,20 +427,13 @@ MainWindow::AchievementCheck(void)
 		numAchieved++;
 	}
 
-	BAlert *achievementAlert;
-
+	AchievementWindow *achievementwin = new AchievementWindow();
+	
 	if (numAchieved > 1) {
```

Comment:
The below does work of course, but if you want to learn now to perfectly do this, that will also guarantee correct localization in any language, study this, for example:
https://git.haiku-os.org/haiku/tree/src/preferences/filetypes/ApplicationTypesWindow.cpp#n349

The title should of course also be B_TRANSLATE()d.

Reply:
I see - thanks for pointing it out.